integration_tests/projects/001_flow_run_with_selectors/fal_scripts/before.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+import io
+import os
+from functools import reduce
+
+model_name = context.current_model.name
+
+output = f"Model name: {model_name}"
+output = output + f"\nStatus: {context.current_model.status}"
```

Comment:
this is weird for me, how does a before see status?

Reply:
this raises questions in general for the before scripts `context` passed in

Reply:
oh I just used what we had before, I dont really even check the content of the file just that it exists.. I can remove the status part.

Reply:
OK, what do you think about

> this raises questions in general for the before scripts `context` passed in